src/com/lits/hw5/Array.java

```diff
@@ -0,0 +1,36 @@
+package com.lits.hw5;
+
+import java.util.Random;
+
+class Array {
```

Comment:
The Arrayrray is an abstraction
not correct

Reply:
don't understood